collectors/python.d.plugin/web_log/web_log.chart.py

```diff
@@ -660,7 +660,7 @@ def find_regex(self, last_line):
                                        r' (?P<bytes_sent>\d+)'
                                        r' (?P<resp_length>\d+)'
                                        r' (?P<resp_time>\d+\.\d+)'
-                                       r' (?P<resp_time_upstream>[\d.-]+) ')
+                                       r' (?P<resp_time_upstream>[\d.-]+)')
```

Comment:
`r' "?(?P<resp_time_upstream>[\d.-]+)(, [\d.]+)*"?')`
it should match `0.01`, `"0.01"`, `0.01, 0.02` and `"0.01, 0.02"`.

Reply:
Since `resp_time_upstream` is the last parameter `"?(?P<resp_time_upstream>[\d.-]+)` is enough, it will match the same but faster

Reply:
if we do not need sum all `resp_time_upstream` ofc

Reply:
Because `resp_time_upstream` should be wrapped with `"`, I think 
`r' "?(?P<resp_time_upstream>[\d.-]+)')` is better.